<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>293</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="reses.qrc">
    <normaloff>:/icons/mainico.ico</normaloff>:/icons/mainico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>206</y>
      <width>412</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>Передача файла</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>17</y>
       <width>400</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="selectFile">
          <property name="toolTip">
           <string>Выбор файла для передачи</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="reses.qrc">
            <normaloff>:/icons/foldico.ico</normaloff>:/icons/foldico.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendFile">
          <property name="toolTip">
           <string>Пtредать выбранный файл</string>
          </property>
          <property name="text">
           <string>Передать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>3</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Статус</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="reses.qrc">:/statuses/offline</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Соединение установлено</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="reses.qrc">:/statuses/online</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>3</y>
      <width>326</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Соединение</string>
    </property>
    <widget class="QTextEdit" name="log">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>15</y>
       <width>232</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>11</y>
       <width>73</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Скорость:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>comboBox_2</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Чётность:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>comboBox_2</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>none</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>odd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>even</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>marker</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>space</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Данных:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>comboBox_2</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_4">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Стоп-биты:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>comboBox_2</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_5">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="tryToConnect">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>100</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="reses.qrc">
      <normaloff>:/picts/conn.ico</normaloff>:/picts/conn.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="tryToDisconnect">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>123</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="reses.qrc">
      <normaloff>:/picts/disconn.ico</normaloff>:/picts/disconn.ico</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>146</y>
      <width>70</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Порт:</string>
    </property>
    <widget class="QRadioButton" name="com1rbt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>52</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>COM1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="com2rbt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>52</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>COM2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="listenBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="reses.qrc">
      <normaloff>:/picts/unlisten.gif</normaloff>:/picts/unlisten.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="reses.qrc"/>
 </resources>
 <connections/>
</ui>
